instances/experimentalApproach/biophysics.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/biophysics",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/ExperimentalApproach",
+  "definition": null,
```

Comment:
@UlrikeS91 I would go with the general definition of wikipedia, and then later explain in description a bit more what specifically is considered to fall under biophysics.

Suggestion (@tgbugs please feel free to adapt and to suggest a more detailed description in addition):
"'Biophysics' covers any experimental approach that is traditionally used in physics, but applied and modified to study biological phenomena. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Biophysics)]"

Reply:
Added your definition 🙂